tools/azsdk-cli/Azure.Sdk.Tools.Cli/CHANGELOG.md

```diff
@@ -1,16 +1,15 @@
 # Release History
 
-## 0.5.8 (Unreleased)
+## 0.5.8 (2025-11-13)
 
 ### Features Added
 
 - Validate package names before adding to release plan
 - Allow forced creation when a release plan already exists
 - Add support for sample translation from one language to another
+- Updated CLI commands hierarchy
```

Comment:
Would it help people reading the changelog if there was a bit more detail about the specific changes in the hierarchy? If I were a customer using the CLI and I was broken by this change this changelog entry would not help me absorb the break -- I'd have to dig into the code in this PR to see what exactly happened.

Reply:
If we changed almost all the commands, all the more reason to be explicit about what happened since it's more likely we will break someone.